modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_portlet.js

```diff
@@ -425,9 +441,18 @@ AUI.add(
 								{
 									after: {
 										success: function() {
-											instance._defineIds(this.get('responseData'));
+											var responseData = this.get('responseData');
+
+											instance._defineIds(responseData);
 
 											instance.savedState = state;
+
+											instance.fire(
+												'autosave',
+												{
+													modifiedDate: responseData.modifiedDate
```

Comment:
@brandizzi, can you take a look at this? I'm worried about the displayed date format and timezone issues.